infrastructure_playbook_repo.md

```diff
@@ -0,0 +1,179 @@
+# [Infrastructure Playbook repository](https://github.com/usegalaxy-eu/infrastructure-playbook)
+
+## Generic introduction
+* This repository contains the [Ansible](https://www.ansible.com/) playbooks, roles, etc for the European Galaxy server. It is used to deploy the infrastructure for the European Galaxy server through [Jenkins](https://build.galaxyproject.eu/job/usegalaxy-eu/). All configurational changes related to the [Galaxy EU](https://usegalaxy.eu) are made through this repository.
+* Galaxy EU compute infrastructure is run on the [BW OpenStack cloud](https://portal.bw-cloud.org/). At the time of writing (21/02/2023) our cloud is of size 8488 VCPUs, 44.6 TB RAM, 162.6 TB storage. Additionally, a few petabytes of [storage](https://github.com/usegalaxy-eu/operations/blob/main/storage.md) is also mounted (NFS) in the cloud.
+* The compute infrastructure (cloud cluster) is configured through [VGCN infrastructure repo](https://github.com/usegalaxy-eu/vgcn-infrastructure) where we define what [cloud images](https://github.com/usegalaxy-eu/vgcn) should be used, the size of the cloud cluster, the number of VMs, the cloud network, the cloud security groups, etc.
+* The cloud is configured through this [infrastructure repo](https://github.com/usegalaxy-eu/infrastructure) using [Terraform](https://www.terraform.io/). The underlying cloud hardware, storage, network, etc are managed by the compute center of the University of Freiburg. For DNS records we use [Amazon's Route53](https://aws.amazon.com/route53/).
```

Comment:
Only the non-worker-nodes and dedicated machines for groupmembers

Reply:
Updated: https://github.com/usegalaxy-eu/operations/pull/22/commits/0a07e911b1736c51a20943595f937ccc219a3e9f